packages/components/src/internal/components/editable/models.ts

```diff
@@ -129,10 +130,11 @@ export class EditorModel
     declare focusValue: List<ValueDescriptor>;
     declare id: string;
     declare isPasting: boolean;
+    declare isSparseSelection: boolean;
     declare rowCount: number;
     declare selectedColIdx: number;
     declare selectedRowIdx: number;
-    declare selectionCells: ImmutableSet<string>;
+    declare selectionCells: string[];
```

Comment:
maybe add a comment about this being a sorted array and where the sorting is done (i.e. `applyEditableGridChangesToModels`)

Reply:
Done.